cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3334,7 +3334,7 @@ TEST(ArrowReadWrite, NestedRequiredOuterOptional) {
     auto arrow_writer_props = ArrowWriterProperties::Builder();
     arrow_writer_props.store_schema();
     if (inner_type->id() == ::arrow::Type::UINT32) {
-      writer_props.version(ParquetVersion::PARQUET_2_4);
+      writer_props.version(ParquetVersion::PARQUET_2_6);
```

Comment:
The default writer_props created above with `WriterProperties::Builder();` will now use 2_6 by default, so potentially this could be removed alltogether